Sources/FluentPostgreSQL/PostgreSQLDatabase+SchemaSupporting.swift

```diff
@@ -23,7 +23,12 @@ extension PostgreSQLDatabase: SchemaSupporting, IndexSupporting {
                     string += " GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY"
                 } else {
                     // not appending!
-                    string = "SERIAL PRIMARY KEY"
+                    switch field.type.type {
+                    case .int2: string = "SMALLSERIAL PRIMARY KEY"
+                    case .int4: string = "SERIAL PRIMARY KEY"
+                    case .int8: string = "BIGSERIAL PRIMARY KEY"
+                    default: fatalError("should be unreachable")
```

Comment:
We should throw a `FluentPostgreSQL` error here just in case new cases are added or the surrounding code changes.

Reply:
The problem is that the containing function doesn't throw. Also, shouldn't this happen only during migrations, at which point this can be recognized and fixed, so this wouldn't happen in production anyway?

Reply:
Hmm I thought that one allowed throwing (did you try adding?). If not, then yeah a `fatalError` is not the end of the world here. However, there should be a nicer message there in case someone hits it.

```swift
fatalError("\(field.type.type) primary key not supported") 
```

something like that ^ maybe